tests/test_modifiers.py

```diff
@@ -86,6 +82,37 @@ def test_inserting_an_empty_if_statement(self, create_instance: Modifiers):
         assert isinstance(value, Modifiers)
         assert any(stmt in modified for stmt in STATEMENTS for modified in value.modified_contents)
 
+    def test_reversing_booleans(self, create_instance: Modifiers):
+        value = create_instance.reverse_booleans()
+
+        assert isinstance(value, Modifiers)
+        assert ("True" not in value.modified_contents[1]) != ("False" not in value.modified_contents[1])
+        assert (value.modified_contents[1].count("True") == 2) != (value.modified_contents[1].count("False") == 2)
+
+    def test_breaking_equals_statement(self, create_instance: Modifiers):
+        value = create_instance.break_equals_statement()
+
+        assert isinstance(value, Modifiers)
+        assert "==" not in value.modified_contents[1]
+        assert " = " in value.modified_contents[1]
+
+    def test_mixing_type_keywords(self, create_instance: Modifiers):
+        value = create_instance.mix_type_keywords()
+
+        assert isinstance(value, Modifiers)
+        assert "str" not in value.modified_contents
+        assert any(value_type in value.modified_contents[0] for value_type in TYPES)
+
+    def test_adding_or_removing_brackets(self, create_instance: Modifiers):
+        value = create_instance.add_or_remove_brackets()
+
+        assert isinstance(value, Modifiers)
+        assert any(
+            value.file_contents[num].count(bracket) != value.modified_contents[num].count(bracket)
+            for bracket in "()[]"
+            for num in range(len(value.file_contents))
+        )
```

Comment:
Could this fail if the number of brackets added are the same as the number of brackets removed?

Reply:
It can make only one change per line so this can't be a case also

Reply:
Can resolve